components/SensorComponent.js

```diff
@@ -1,96 +1,92 @@
 import React, { useState, useEffect } from "react";
 import { Accelerometer } from "expo-sensors";
 import styled from "styled-components/native";
+import { View, Text, TouchableOpacity } from "react-native";
 
-// ==========================
-// = Functions
 const isShaking = (data) => {
-  // x,y,z CAN be negative, force is directional
-  // We take the absolute value and add them together
-  // This gives us the total combined force on the device
   const totalForce = Math.abs(data.x) + Math.abs(data.y) + Math.abs(data.z);
-
-  // If this force exceeds some threshold, return true, otherwise false
-  // Increase this threshold if you need your user to shake harder
   return totalForce > 1.78;
 };
 
-// ==========================
-// = Styled components
-const ShakeView = styled.View`
+const Container = styled.View`
   display: flex;
-  flex-direction: column;
+  text-align: center;
+  margin-bottom: 20px;
+  overflow-wrap:break-word;
+  margin-left: 20px;
+  margin-right:20px;
```

Comment:
An alternative to writing out each margin is to group them in the order top, right, bottom, left (clockwise). If you want zero margin, you just write 0 no need to write units.
```
// margins can be written margin: top right bottom left
// which in your case looks like below:

margin: 0 20px 20px 20px;
```
But the way you've done it also works perfectly fine!